File: src/com/xwkj/form/domain/Feedback.java
package com.xwkj.form.domain;

import java.io.Serializable;
import java.util.Date;

public class Feedback implements Serializable {
	private static final long serialVersionUID = 3786109151386805918L;
	
	private String fid;
	private String name;
	private String telephone;
	private String address;
	private String device;
	private String message;
	private Date createDate;
	
	public String getFid() {
		return fid;
	}
	public void setFid(String fid) {
		this.fid = fid;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getTelephone() {
		return telephone;
	}
	public void setTelephone(String telephone) {
		this.telephone = telephone;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getDevice() {
		return device;
	}
	public void setDevice(String device) {
		this.device = device;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
	public Date getCreateDate() {
		return createDate;
	}
	public void setCreateDate(Date createDate) {
		this.createDate = createDate;
	}

}

File: WebContent/static/js/feedback.js
$(document).ready(function() {
	$("#feedback-submit").click(function() {
		var name=$("#feedback-name").val();
		var telephone=$("#feedback-telephone").val();
		var address=$("#feedback-address").val();
		var device=$("#feedback-device").val();
		var message=$("#feedback-message").val();
		if(name==""||telephone==""||address==""||device=="") {
			alert("请填写所有带星号的必填项目！");
			return;
		}
		FeedbackManager.add(name, telephone, address, device, message, function(fid) {
			if(fid) {
				alert("提交成功");
				location.href="index.html";
			}
		});
	});
});